setup.py

```diff
@@ -67,6 +72,14 @@ def run(self):
 
 if __name__ == "__main__":
     write_version_file()
+    NAME_ARG = "--override-name"
+    if NAME_ARG in sys.argv:
+        idx = sys.argv.index(NAME_ARG)
+        name = sys.argv.pop(idx + 1)
+        sys.argv.pop(idx)
+        is_nightly = "nightly" in name
+        if is_nightly:
+            version = get_nightly_version() 
```

Comment:
I wonder if we'd need a nightly package yet?

Per the command in `build.py`, this code section would introduce another package named `PiPPy-nightly` (case-sensitive, defined in `build.py`), as compared to `pippy` (defined in this file).